Bit-Manipulation/IsPowerofFour.js

```diff
@@ -0,0 +1,24 @@
+/*
+    author: @dev-madhurendra
+
+    This script will check whether the given
+    number is a power of four or not.
+
+    A number will be power of four if and only if there is a 1 in the binary
```

Comment:
This is wrong. `10` (2) is not a power of 4. A number is a power of four if there is a single set bit followed by an even number of zeroes. What really makes this work is the `n % 3 === 1`, which rules out powers of 2 which are not powers of 4.